query-connector/src/app/database-service.ts

```diff
@@ -117,43 +101,18 @@ export const executeDefaultQueryCreation = async () => {
  */
 export const mapQueryRowsToValueSets = async (rows: QueryResultRow[]) => {
   // Create groupings of rows (each of which is a single Concept) by their ValueSet ID
-  const vsIdGroupedRows = rows.reduce((conceptsByVSId, r) => {
-    if (!(r["valueset_id"] in conceptsByVSId)) {
-      conceptsByVSId[r["valueset_id"]] = [];
-    }
-    conceptsByVSId[r["valueset_id"]].push(r);
-    return conceptsByVSId;
-  }, {});
-
-  // Each "prop" of the struct is now a ValueSet ID
-  // Iterate over them to create formal Concept Groups attached to a formal VS
-  const valueSets = Object.keys(vsIdGroupedRows).map((vsID) => {
-    const conceptGroup: QueryResultRow[] = vsIdGroupedRows[vsID];
-    const valueSet: ValueSet = {
-      valueSetId: conceptGroup[0]["valueset_id"],
-      valueSetVersion: conceptGroup[0]["version"],
-      valueSetName: conceptGroup[0]["valueset_name"],
-      // External ID might not be defined for user-defined valuesets
-      valueSetExternalId: conceptGroup[0]["valueset_external_id"]
```

Comment:
I think in the case where those values aren't defined by the user, they'd be null in the database for the relevant custom valueset. When the "save the configured query" function fires, we'd have (yet to be written) code that would reference those rows in the valueset table, they'd pick up the null values  into the passed in param for `rows` on 102 and eventually set on 109. 

In other words, since we're now setting valueset data straight from the DB (not including some maps/reduces for dimensionality), what's happening on 143-144 and 138-139 should just get handled by how we save the fields in those columns on custom valuesets. 

Let me know if I'm missing something though.